client/js/libs/handlebars/friendlydate.js

```diff
@@ -4,10 +4,10 @@ const moment = require("moment");
 
 module.exports = function(time) {
 	// See http://momentjs.com/docs/#/displaying/calendar-time/
-	return moment(new Date(time)).calendar(null, {
+	return moment(time).calendar(null, {
 		sameDay: "[Today]",
 		lastDay: "[Yesterday]",
-		lastWeek: "L", // Locale
-		sameElse: "L"
+		lastWeek: "MMMM DD, YYYY",
+		sameElse: "MMMM DD, YYYY"
```

Comment:
Ah ah, calling me uncultured much? :D

I would be arguing if the entire app was consistent, but since we are not even consistent with one format (I thought we were, with the American one, but didn't realize we had `16:30`, I'm just too vested in this format to see it!), then not so much.

I do think we'll need to bring back this discussion when we localize the whole client, but until then, I'm convinced (especially with the conversion from `02-04` into `2 May`. Or is it `4 February`? :D)

However, since we're here, let's go for `DD MMMM YYYY`. It feels more balanced (no need for a comma to visually separate day and year), it represents 10 times more populations than the alternative according to your link, etc. (it's also the standard date format for W3C standards :D)